### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Recillah-Khamala**
### **Hello Rails Back end**

Hello CR

In this project, I have implemented the below requirements-:

- Created a new Rails API app called 'hello-rails-back-end'.
- Initialized project with Git.
- Made sure that the project has a Postgres database set up.
- Created a table for storing your messages and created 5 different greetings.
- Created an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+- [ ] **SignUp and Login**
+- [ ] **Authentication and Authorization**
+- [ ] **Documentation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FRONT-END REPOSITORY -->
+## 💻 Front-end Repository <a name="frontend-repo"></a>
+
+- [Click Here](https://github.com/Recillah-Khamala/hello-react-frontend)
```
Comment (mckent05) [inline README.md]: - You have done a good job so far, but please confirm that the link to your front-end repo is correct 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Recillah-Khamala, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍
- Your backend application works as expected 👍 👍  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Recillah-Khamala) [reply README.md]: done

**Review by julie-ify — APPROVED** [review]
Hi @Recillah-Khamala,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**